app/controllers/omniauth_controller.rb

```diff
@@ -1,6 +1,5 @@
 class OmniauthController < Devise::OmniauthCallbacksController
   skip_before_action :verify_authenticity_token, only: [:tra_openid_connect]
-  before_action :remove_from_get_an_identity_pilot, only: :tra_openid_connect, if: :tra_response_excludes_trn?
```

Comment:
The concept of the pilot is entirely removed in this PR, users are always tied to the GAI account they logged in via. The fallback for no TRN being present is to now introduce the qualified_teacher_check after the oauth login as the first question, then move right into the rest of the flow.